modules/search/instant-search/components/search-results.jsx

```diff
@@ -23,33 +23,39 @@ class SearchResults extends Component {
 	}
 
 	render() {
-		const { results = [], query, total = 0, corrected_query = false } = this.props;
+		const { results = [], query, total = 0, corrected_query = false, loading = false } = this.props;
 		if ( query === '' ) {
 			return <div className="jetpack-instant-search__search-results" />;
 		}
 		if ( total === 0 ) {
 			return (
 				<div className="jetpack-instant-search__search-results">
 					<div>
-						<h3>{ sprintf( __( 'No Results.' ), query ) }</h3>
+						<h3>{ sprintf( __( 'No Results.', 'jetpack' ), query ) }</h3>
 					</div>
 				</div>
 			);
 		}
+		const num = new Intl.NumberFormat().format( total );
```

Comment:
Heads up: [`Intl.NumberFormat` isn't supported by IE 6-10](https://caniuse.com/#feat=mdn-javascript_builtins_intl_numberformat). I don't think Jetpack provides a polyfill for this function automatically, either. 

[Number.prototype.toLocaleString](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toLocaleString) might be a good alternative here.

Reply:
No problem: Jetpack and wpcom support down to IE 11. 😉

Reply:
😆